tests/functional/d/deprecated/deprecated_relative_import/subpackage/dot_dot_relative_import.rc

```diff
@@ -0,0 +1,2 @@
+[testoptions]
+min_pyver=3.2
```

Comment:
This isn't required because the min supported version is python 3.6 anyway. (I'm saying this so you don't have to do it next time, no need to change that :) )

Reply:
Thank you good to know. I saw that other tests include even older version of python than supported by pylint, so I considered it as a practise to put it there...

Reply:
Yes, we should probably clean up when we stop supporting Python 3.6, because this is confusing. I don't think anyone will fork to support a lower version of Python than us anyway. I'm going to open an issue.